edward/inferences/wgan_inference.py

```diff
@@ -90,10 +93,14 @@ def build_loss_and_gradients(self, var_list):
                                      list(range(1, gradients.shape.ndims))))
       penalty = self.penalty * tf.reduce_mean(tf.square(slopes - 1.0))
 
+    reg_terms_d = tf.losses.get_regularization_losses(scope="Disc")
+    reg_terms_all = tf.losses.get_regularization_losses()
```

Comment:
Any reason this one  and GAnInference uses all but BiGAN's uses `scope="Gen"`?

Reply:
I went by how variable lists are created in the respective files. For GAN and WGAN, `scope="Gen"` is not used for creating `var_list` so I assumed it wasn't there.

Reply:
That makes sense. Merging now. All looks great.